ipld/src/util.rs

```diff
@@ -75,46 +87,82 @@ where
     let bytes = load_block(root).await?;
     let ipld = from_slice(&bytes)?;
 
-    traverse_ipld_links_hash(walked, load_block, &ipld).await?;
+    traverse_ipld_links_hash(walked, load_block, &ipld, on_inserted).await?;
 
     Ok(())
 }
 
+lazy_static! {
+    pub static ref WALK_SNAPSHOT_PROGRESS_EXPORT: Arc<(AtomicU64, AtomicU64)> =
```

Comment:
Let's have a nice, named alias for this. We are using this construct in several places here.

Reply:
It would also be nice to avoid using `lazy_static!`.

Reply:
@lemmih could u elaborate? Do u suggest switch to sth else like `OnceCell` or avoid singleton trackers?

Reply:
@LesnyRumcajs Fixed